x-pack/plugins/file_upload/public/importer/importer.ts

```diff
@@ -22,7 +22,7 @@ import { CreateDocsResponse, IImporter, ImportResults } from './types';
 
 const CHUNK_SIZE = 5000;
 const MAX_CHUNK_CHAR_COUNT = 1000000;
-const IMPORT_RETRIES = 5;
+export const IMPORT_RETRIES = 5;
 const STRING_CHUNKS_MB = 100;
```

Comment:
This is only used once down below and is multiplied * `MB`. Why not just use `MAX_FILE_SIZE_BYTES`?

Reply:
I am trying not to touch logic for CSV import with these changes. I am not sure why `STRING_CHUNKS_MB` was originally defined and why its defined separate to `MAX_FILE_SIZE_BYTES` but I would rather not change it in this pull request since its not related to these changes.